Week3/homework/js-exercises/4-shopping.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+let groceryItemArray = ["Milk", "Banana"];
+
+function addToShoppingCart(item){
+   if(groceryItemArray.length > 2)
+     {groceryItemArray.shift();}
+   groceryItemArray.push(item);
```

Comment:
👍 for logic but indentation problems here as well. Makes it hard to read 🙁

Reply:
In this exercise actually I wanted to push a random item from another array of products, I used math.random, it generated the same number over and over again and after certain point, there was an array with only one type of product. Then I did this relatively easy one.